src/main/java/mate/jdbc/service/DriverServiceImpl.java

```diff
@@ -0,0 +1,48 @@
+package mate.jdbc.service;
+
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.dao.DriverDao;
+import mate.jdbc.lib.Inject;
+import mate.jdbc.lib.Service;
+import mate.jdbc.model.Driver;
+
+@Service
+public class DriverServiceImpl implements DriverService {
+    @Inject
+    private DriverDao driverDao;
+
+    @Override
+    public Driver create(Driver driver) {
+        Optional<Driver> driverByLicenseNumber = 
```

Comment:
why do you need this check? for this method it is enough to:
return driverDao.create(driver);

Reply:
@ kozhukhovsky, explanation to check in 22-23 line in 'create' method. I think It was logical that the license numbers need to be unique and personal.

Reply:
@kateryna-mykh That's a good way of thinking about it, but it's not necessary in this particular assignment.